docs/core/platform/significant_change.md

```diff
@@ -8,17 +8,18 @@ This support is added by creating a `significant_change.py` platform file with a
 
 ```python
 from typing import Any, Optional
-from homeassistant.core import callback
+from homeassistant.core import HomeAssistant, callback
 
+# pylint: disable=unused-argument
 @callback
 def async_check_significant_change(
     hass: HomeAssistant,
     old_state: str,
     old_attrs: dict,
     new_state: str,
     new_attrs: dict,
-    **kwargs: Any,
-) -> Optional[bool]
+    **kwargs: Any
```

Comment:
This change is actually incorrect, we format use `black`.

Reply:
Removed trailing comma